src/lib/containers/CardContainer.tsx

```diff
@@ -27,11 +28,26 @@ export type CardContainerProps = {
   isHeader?: boolean
   isAlignToLeftNav?: boolean
   title?: string
-  facetAliases?: Record<string, string>
   isLoading?: boolean
   unitDescription?: string
 } & CardConfiguration
 
+function Card(props: { propsToPass: any; type: string }) {
```

Comment:
This was defined in the CardContainer component below, which probably resulted in excessive re-renders